plugin/src/utils/report.ts

```diff
@@ -15,49 +17,58 @@ export const CODES = {
   SourcePluginCodeError: `112003`,
 }
 
-export const ERROR_MAP = {
+interface IErrorContext {
+  sourceMessage: string
+}
+interface IErrorMap {
+  [code: string]: IErrorMapEntry
+}
+
+const getErrorText = (context: IErrorContext): string => context.sourceMessage
+
+export const ERROR_MAP: IErrorMap = {
   [CODES.WordPressFilters]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `USER`,
   },
   [CODES.BadResponse]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `USER`,
   },
   [CODES.RequestDenied]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `USER`,
   },
   [CODES.Authentication]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `USER`,
   },
   [CODES.Timeout]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `USER`,
   },
   [CODES.RemoteGraphQLError]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `THIRD_PARTY`,
   },
   [CODES.MissingAppendedPath]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `THIRD_PARTY`,
   },
   [CODES.SourcePluginCodeError]: {
-    text: (context) => context.sourceMessage,
+    text: getErrorText,
     level: `ERROR`,
     category: `SYSTEM`,
   },
   [CODES.WordPress500ishError]: {
-    text: (context) => context.sourceMessage,
+    text: (context: IErrorContext): string => context.sourceMessage,
```

Comment:
Just adding a reminder in my review :) we should probably use the `getErrorText` function on this one as well. nice clean up here!

Reply:
ah yes, i noticed that, which is what i referred to here:

https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/pull/332#discussion_r542813510

happened because of an upstream rebase with the primary branch